### PR TITLE
use spec data in pretty printing

### PR DESCRIPTION
**Pull request by mmoskal**

### CODE REVIEW

File: src/dom/utils.ts [hunk 1]
```diff
@@ -21,6 +21,10 @@ export function delay<T>(millis: number, value?: T): Promise<T> {
     return new Promise((resolve) => setTimeout(() => resolve(value), millis))
 }
 
+export function values<T>(m: SMap<T>): T[] {
```
Comment (pelikhan) [inline src/dom/utils.ts:24]: Object values?

Comment (mmoskal) [reply src/dom/utils.ts:24]: I must be stuck in the past

Comment (pelikhan) [reply src/dom/utils.ts:24]: also is there a better way to consume that spec repo? right I've just copied a build over but it's not great...

File: src/dom/specdata.ts [hunk 2]
```diff
@@ -0,0 +1,1970 @@
+export const serviceSpecifications: jdspec.SMap<jdspec.ServiceSpec> =
```
Comment (pelikhan) [inline src/dom/specdata.ts:1]: for the purpose of gatsby, it would be nice to have a .json drop of the spec.

**Comment by pelikhan** [comment]
The gatsby stuff likes it better if the services were an array rather than a map.

**Pull request merged**